projects/backend/controllers/crossword.ts

```diff
@@ -0,0 +1,38 @@
+import { Request, Response } from 'express'
+import { Issue } from '../common'
+import { lastModified } from '../lastModified'
+import fetch from 'node-fetch'
+import { Crossword } from '@guardian/capi-ts'
+import moment from 'moment'
+import console = require('console');
+
+
+const crosswordDateParamFormat: string = 'YYYY-MM-DD'
+const crosswordApiKey: string | undefined = process.env.CROSSWORD_API_KEY || undefined
```

Comment:
If you want to test this locally yourselves let me know and I can provide you with the API key.